client/blocks/importer/wordpress/utils.js

```diff
@@ -43,3 +43,11 @@ export const getAcceptedAssistedFreeMigration = () => {
 		sessionStorage?.getItem( 'wpcom_import_migration_assistance_accepted' )
 	);
 };
+export const setMigrationAssistanceAccepted = () =>
+	ignoreFatalsForSessionStorage( () =>
+		sessionStorage?.setItem( 'wpcom_import_migration_assistance_accepted', true )
```

Comment:
Can we use a constant for this and the other two functions that use this? Like they do https://github.com/Automattic/wp-calypso/pull/89917/files#diff-66a2d71fb0d2016f0885b31670a7a7b1a0246456b01844bf4a248e06954764edR38